store/rootmulti/store.go

```diff
@@ -609,7 +621,8 @@ func (si storeInfo) Hash() []byte {
 		panic(err)
 	}
 
-	return hasher.Sum(nil)
+	hash := hasher.Sum(nil)
```

Comment:
Why this change?

Reply:
Missed reverting this change during debugging